Toolchain/Patches/binutils.patch

```diff
@@ -1,24 +1,20 @@
-From e80a89e9446501bcaf42dc8ade8ff8a1cd0458bb Mon Sep 17 00:00:00 2001
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
This hash looks a bit funny,
is that really correct?

Reply:
That is correct, although it's no longer meant to represent a hash.

The reason is that the commit hashes that we exported those patches from generally don't have much meaning to us (as we are rarely pulling in upstream commits as patches, and if so not for an extended time). As a result, we have recently started to export patches for ports without numbers, commit hashes or Git version numbers (#14199) to avoid noise when regenerating patches.